cosmic-applet-power/i18n/nl/cosmic_applet_power.ftl

```diff
@@ -0,0 +1,21 @@
+power = Stroom
+settings = Instellingen...
+lock-screen = Vergrendel Scherm
+lock-screen-shortcut = Super + Escape
+log-out = Uit loggen
+log-out-shortcut = Ctrl + Alt + Delete
+suspend = Slaapstand
```

Comment:
technically "slaapstand" would be sleep-mode, but in Dutch these are often used interchangably